MaterialSkin/HTML/material/html/css/desktop.css

```diff
@@ -11,6 +11,16 @@
     --bottom-toolbar-height: 80px;
 }
 
+
```

Comment:
Should there not be a similar entry for mobile.css?

Reply:
Good point. I will add.